src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3703,8 +3703,9 @@ def list_vmss_instance_public_ips(cmd, resource_group_name, vm_scale_set_name):
 def reimage_vmss(cmd, resource_group_name, vm_scale_set_name, instance_id=None, no_wait=False):
     client = _compute_client_factory(cmd.cli_ctx)
     if instance_id:
-        return sdk_no_wait(no_wait, client.virtual_machine_scale_set_vms.begin_reimage,
-                           resource_group_name, vm_scale_set_name, instance_id)
+        for instance in instance_id:
+            sdk_no_wait(no_wait, client.virtual_machine_scale_set_vms.begin_reimage,
+                        resource_group_name, vm_scale_set_name, instance)
```

Comment:
The SDK method `client.virtual_machine_scale_set_vms.begin_reimage` itself does not support the batch operation, right?